examples/get_iss_data.py

```diff
@@ -133,14 +80,44 @@ def format_data(input_dir, output_dir, d):
         input_dir += "ExampleInSituSequencing/"
         print("Using data in : {}".format(input_dir))
 
-    image_stack = build_hybridization_stack(input_dir)
-    image_stack_name = "hybridization.json"
-    Writer.write_to_path(
-        image_stack,
-        os.path.join(output_dir, image_stack_name),
-        pretty=True,
-        tile_opener=tile_opener,
-        tile_writer=tile_writer)
+    def add_codebook(experiment_json_doc):
+        experiment_json_doc['codebook'] = "codebook.json"
+
+        # TODO: (ttung) remove the following unholy hacks.  this is because we want to point at a tileset rather than
+        # a collection.
+        experiment_json_doc['hybridization_images'] = "hybridization-fov_000.json"
+        experiment_json_doc['auxiliary_images']['nuclei'] = "nuclei-fov_000.json"
+        experiment_json_doc['auxiliary_images']['dots'] = "dots-fov_000.json"
+        return experiment_json_doc
+
+    write_experiment_json(
+        output_dir,
+        1,
+        {
+            Indices.HYB: 4,
+            Indices.CH: 4,
+            Indices.Z: 1,
+        },
+        {
+            'nuclei': {
+                Indices.HYB: 1,
+                Indices.CH: 1,
+                Indices.Z: 1,
+            },
+            'dots': {
+                Indices.HYB: 1,
+                Indices.CH: 1,
+                Indices.Z: 1,
+            }
+        },
+        hyb_image_fetcher=HybridizationImageFetcher(input_dir),
+        aux_image_fetcher={
+            'nuclei': AuxImageFetcher(os.path.join(input_dir, "DO", "c1.TIF")),
```

Comment:
Could you use keyword arguments here? Might make these parameter choices clearer to someone reading the code.

Reply:
Yea, I meant:
```
'nuclei': AuxImageFetcher(os.path.join(
    input_dir, 
    whatever_this_param_is="DO", 
    i_am_probably_an_informative_param_name_too="c1.TIF")
), ...
```

Reply:
Man when did I do this PR? I must have been half asleep. Please ignore. I somehow thought that DO and c1.TIF were arguments to `AuxImageFetcher`, not paths to join.

Reply:
🤦‍♂️